src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -45,7 +45,8 @@ class BasePaymentsPage extends React.Component {
             selectedPaymentMethod: {},
             formattedSelectedPaymentMethod: {},
             anchorPositionTop: 0,
-            anchorPositionLeft: 0,
+            anchorPositionBottom: 0,
+            anchorPositionRigth: 0,
```

Comment:
```suggestion
            anchorPositionRight: 0,
```

Can you also correct the typo at other places?